cmd/crio/main.go

```diff
@@ -269,6 +269,26 @@ func main() {
 			return err
 		}
 
+		// Print the current CLI flags.
+		for _, flagName := range c.FlagNames() {
+			flagValue := c.Value(flagName)
+			// Turn a multi-value flag into a single command-separated list
+			// of arguments, then wrap into a slice so that %v does it work
+			// for us when rendering a slice type in the output.
+			if _, ok := flagValue.(cli.StringSlice); ok {
+				flagValue = []string{strings.Join(c.StringSlice(flagName), ",")}
+			}
+			logrus.Infof("FLAG: --%s=\"%v\"\n", flagName, flagValue)
+		}
+
+		// Print the current configuration.
+		tomlConfig, err := config.ToString()
+		if err != nil {
+			logrus.Errorf("Unable to print current configuration: %v", err)
+		} else {
+			logrus.Infof("Current CRI-O configuration:\n%s", tomlConfig)
+		}
```

Comment:
To clarify before we wrap things up here: do we want to print configuration and command-line flags at the "info" level (default) or "debug"? Some might argue that the rather verbose nature of the configuration file being printed would be better suited for the "debug" log level.

@LenkaSeg and @cri-o/cri-o-maintainers, though?

Reply:
Considering that it's adding quite a voluminous output, maybe it would be cleaner to have the config print in a debug level. Unless there's a necessity to have the config captured for every cri-o run.

Reply:
I think it's useful enough to have at info. It's not regularly verbose, since it only prints once at startup. I think that's a fine granularity to get the gain of immediate insight into the configuration

Reply:
OK. We have a consensus here. Thank you!